riot/riot.py

```diff
@@ -86,6 +86,15 @@ def version(self) -> str:
         version = output.decode().strip().split(" ")[1]
         return version
 
+    @functools.lru_cache()
+    def version_info(self) -> t.Tuple[int, int, int]:
+        version_head = "".join(
+            list(itertools.takewhile(lambda _: _.isdigit() or _ == ".", self.version()))
```

Comment:
Can we use something like `python -c 'import sys; print("%s.%s.%s" % (sys.version_info.major, sys.version_info.minor, sys.version_info.micro))'` in  the`version()` and something like that here?

Sounds safer than parsing strings.

Reply:
sounds good! on it!